locale/en/docs/guides/simple-profiling.md

```diff
@@ -21,8 +21,9 @@ code-creation,Stub,2,0x2d5000a33e00,507,"NumberToStringStub"
 ```
 
 In the past, you needed the V8 source code to be able to interpret the ticks.
-Luckily, tools have recently been introduced into Node.js 4.4.0 that facilitate
-the consumption of this information without separately building V8 from source.
+Luckily, tools have been introduced into Node.js since v4.4.0 that 
```

Comment:
Before the release of Node.js 8.0.0, we decided that we would not use `v` when referring to Node.js versions. This was to avoid confusion as to whether `v8` referred to Node.js 8.0.0 or Google's V8 JavaScript engine. So, with that in mind, and also some rewriting for simplicity and active voice:

```suggestion
Node.js 4.4.0 introduce tools that
```

Reply:
I’ve also read about this in the [Node.js v8.0.0 release post](https://nodejs.org/en/blog/release/v8.0.0/#long-term-support), however, all of the [recent release posts](https://nodejs.org/en/blog/) still use the `v` prefix in the title. Is the prefixless version still preferred?

Reply:
Yeah, the blogs are awfully problematic in terms of following style. (Another thing they do is use "Node" where "Node.js" is preferred.) There's probably a boilerplate-generating tool that needs to be updated.

Reply:
@Trott：Can you tell me where the tool is so as to let me update it when free? Thanks anyway!